Logger/BaseLogger.cs

```diff
@@ -1,7 +1,12 @@
-﻿namespace Logger;
+﻿using System;
 
-public abstract class BaseLogger
+namespace Logger
 {
-    public abstract void Log(LogLevel logLevel, string message);
+    
+    public abstract class BaseLogger
+    {
+        public  static string Name { get; set; } = string.Empty;
```

Comment:
Not a big deal, but should be `ClassName`. Be sure to stick with instructions in prompt.
```suggestion
        public  static string ClassName{ get; set; } = string.Empty;
```